jabgui/src/main/java/org/jabref/gui/walkthrough/WalkthroughAction.java

```diff
@@ -307,7 +308,7 @@ private Walkthrough createPdfLinkWalkthrough() {
                 .highlight(HighlightEffect.BACKDROP_HIGHLIGHT)
                 .build();
 
-        return new Walkthrough(step1, step2, step3, step4, step5, step6, step7, step8, step9, step10, step11, step12, step13, step14, step15, step16, step17, step18);
+        return new Walkthrough(Injector.instantiateModelOrService(StateManager.class), step1, step2, step3, step4, step5, step6, step7, step8, step9, step10, step11, step12, step13, step14, step15, step16, step17, step18);
```

Comment:
Just thinking: Maybe the long list of steps could be also be refactored to some flow style. Might be better readable and you get rid of the endless step list. Would have to implement LinkedList to ensure correct order i think.
e.g.
```java
Walkthrough myWalkthrough = Walkthrough.create(stateManager);
myWalkthrough.addStep(WalkthroughStep.panel(...)...).build();
myWalkthrough.addStep(...)
...
return myWalkthrough.build();
```